docs/services.md

```
@@ -5,10 +5,18 @@ Completes transactions
-TO CONSIDER: Migrate to volumes, no bind mounts: for consistency
+TO CONSIDER: Migrate to volumes, no bind mounts: for consistency.  (I think I used bind to allow easier access to the log data for debugging)
 
 Use systemctl to trigger runs on linux.
 Use launchd to trigger runs on macos.
 
+prodtest: currently running every 4 hours using launchd -> docker compose up
+ - see tx-processor/docker/com.thecoin.txprocessor.plist
+prod: currently running every 1 hours using systemd -> docker compose up
+ - see tx-processor/docker/txprocessor.service & timer
+
+ ### Logging
```

markdownlint-cli2 (0.23.2)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)